testnets/daodiseotestnet/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "daodiseotestnet",
+  "assets": [
+    {
+      "description": "The native token of Daodiseo Testnet",
+      "denom_units": [
+        {
+          "denom": "uodis",
+          "exponent": 0
+        },
+        {
+          "denom": "ODIS",
+          "exponent": 6
+        }
+      ],
+      "base": "uodis",
+      "name": "Daodiseo",
+      "display": "ODIS",
+      "symbol": "ODIS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/ywil23/chain-registry/refs/heads/master/testnets/daodiseotestnet/images/odiseo.png"
+      }
```

Comment:
Please remove one of the duplicate uploaded images. I recommend keeping the image in the testnets/daodiseotestnet/images/ directory for now, and then it can be relocated to a mainnet directory once that's registered.